src/Storage/Doctrine/PhpcrOdm/ContentTypeDriver.php

```diff
@@ -1,127 +0,0 @@
-<?php
-
-namespace Psi\Component\ContentType\Storage\Doctrine\PhpcrOdm;
-
-use Doctrine\Common\Persistence\Mapping\ClassMetadata;
-use Doctrine\Common\Persistence\Mapping\Driver\MappingDriver;
-use Psi\Component\ContentType\FieldRegistry;
-use Psi\Component\ContentType\Mapping\CompoundMapping;
-use Psi\Component\ContentType\MappingRegistry;
-use Psi\Component\ContentType\MappingResolver;
-
-/**
- * The content type driver will take care of automatically mapping objects that
- * are required by content type fields (a CT field might use an Image model, or
- * a FormConfig object for example). This driver will automatically handle the
- * mapping for these fields.
- */
-class ContentTypeDriver implements MappingDriver
```

Comment:
Will delegate this responsiblity to the user - to do this properly would require external mapping.